### PR TITLE
Featue: Show all questions if no filters

### PR DESCRIPTION
**Pull request by maxplt**
# Jira Issue: [IP-154](https://hedvig.atlassian.net/jira/software/projects/IP/boards/19?selectedIssue=IP-154)

## What?
Show all questions if no filters selected

## Video

https://user-images.githubusercontent.com/89198006/140009866-3e54c745-fee3-4251-a1ae-03f2b1fcf1cb.mov

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

**Review by rasmusguterstam — CHANGES_REQUESTED** [review]
Nothing to add on the slim implementation haha, but can we add this to conversations as well? For consistency

File: src/pages/conversations/ConversationsPage.tsx [hunk 1]
```diff
@@ -51,17 +52,25 @@ const ConversationsPage: React.FC<RouteComponentProps<{
     [],
   )
 
+  console.log(filters)
```
Comment (rasmusguterstam) [inline src/pages/conversations/ConversationsPage.tsx:55]: Naaah 🔪

File: src/pages/conversations/ConversationsPage.tsx [hunk 2]
```diff
@@ -51,17 +52,25 @@ const ConversationsPage: React.FC<RouteComponentProps<{
...
+      setFilteredGroups(
+        questionGroups
+          .filter(doMemberGroupFilter(numberMemberGroups)(filters))
+          .filter(doMarketFilter(filters))
+          .filter(doClaimFilter(filters)),
+      )
+    } else {
+      setFilteredGroups([...questionGroups])
+    }
+  }, [filters])
```
Comment (rasmusguterstam) [inline src/pages/conversations/ConversationsPage.tsx:73]: Why not a memo?

Comment (maxplt) [reply src/pages/conversations/ConversationsPage.tsx]: I did mistake. Returned memo back

**Review by rasmusguterstam — APPROVED** [review]

**Pull request merged**